src/main/java/com/feelfreecincinncati/FeelfreecincinnatiApplication.java

```diff
@@ -1,12 +1,16 @@
 package com.feelfreecincinncati;
 
+import org.joda.time.LocalTime;
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class FeelfreecincinnatiApplication {
 
 	public static void main(String[] args) {
+		LocalTime currentTime = new LocalTime();
+		System.out.println("The time is now: " + currentTime); // use this to know the current time
```

Comment:
A couple of things here... 1) no one is going to see System.out.println.  It would be nice if this were a more meaningful contribution to the outcomes.  2) Avoid adding rationale/justification in source code comments.   Save that discussion for the pull request, where I am writing this comment.  That allows you to have a back and forth conversation without cluttering up the codebase.